src/FirestoreCollection.js

```diff
@@ -29,6 +30,20 @@ class FirestoreCollection extends Component {
   }
 
   componentWillUnmount() {
+    this.handleUnsubscribe();
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (!isEqual(nextProps, this.props)) {
```

Comment:
@damonbauer - Since we're already testing for each individual prop in the tests, we can just do the same here and remove the dependency on `lodash`.